src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
Collapsing imports is against our coding standard. Make sure you set IntelliJ to not do this: Preferences > Editor > Code Style > Java > Imports tab set Class count to use import with '*'